test/deprecated/IO/iohints.chpl

```diff
@@ -0,0 +1,27 @@
+use IO;
+use CTypes;
+
+var h : iohints;
+h = IOHINT_RANDOM;
+h = IOHINT_SEQUENTIAL;
+h = IOHINT_CACHED;
+h = IOHINT_PARALLEL;
+h = IOHINT_NONE;
+
+const ds = defaultIOStyle();
+var f, c;
+
+f = open("./iohints.chpl", iomode.r, hints = h, ds); f.close();
```

Comment:
Since we're not going to have a version of the already deprecated function with the new hints type, I think you'll be able to remove the above four lines from your test (and the ones corresponding to the other functions)